bucket/gpac.json

```diff
@@ -0,0 +1,30 @@
+{
+  "description": "GPAC is an Open Source multimedia framework.",
+  "homepage": "https://gpac.wp.imt.fr/",
+  "version": "0.7.0",
+  "license": {
```

Comment:
URL is not needed for valid SPDX identifier. I guess it's **only** version, since their license is pointing to first version of license.
```suggestion
    "license": "LGPL-2.1-only",
```